### PR TITLE
fix route return

### PR DESCRIPTION
**Pull request by meguiraun**
same as before :)

### CODE REVIEW

File: mxcube3/routes/beamline.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def set_func():
         args = request.get_json()
         adapter = app.mxcubecore.get_adapter(obj.lower())
         adapter.execute_command(cmd_name, args)
-        return make_response("{}", 200)
+        return "Command executed successfull"
```
Comment (fabcor-maxiv) [inline mxcube3/routes/beamline.py:86]: successfully

Comment (fabcor-maxiv) [reply mxcube3/routes/beamline.py:86]: Are there cases where make_response returns None? When? Why?

Comment (meguiraun) [reply mxcube3/routes/beamline.py:86]: seems so, all the calls in this method, no clue why

**Comment by marcus-oscarsson** [comment]
Very nice, ill try to get the build working again after this dependency mess and then Ill try to rebase

File: mxcube3/routes/beamline.py (hunk 1 above)
Comment (marcus-oscarsson) [reply mxcube3/routes/beamline.py:86]: hm, odd

**Pull request merged**